frontend/server/src/Controllers/Course.php

```diff
@@ -2058,7 +2058,18 @@ public static function apiStudentProgress(\OmegaUp\Request $r): array {
                 'assignmentNotFound'
             );
         }
+        return [
+            'problems' => self::getProblemsBySelectedAssignment(
+                $assignment,
+                $resolvedIdentity
+            ),
+        ];
+    }
 
+    private static function getProblemsBySelectedAssignment(
+        $assignment,
+        $resolvedIdentity
```

Comment:
Mismo caso que el comentario anterior, hay que agregar el tipo correcto:
```suggestion
        \OmegaUp\DAO\VO\Identities $resolvedIdentity
```